glitchdotcom/Glitch-Community
# fix visibility of leave project button

## Links
* https://various-dart.glitch.me
* https://app.clubhouse.io/glitch/story/7887/leave-project-button-shown-incorrectly

## Changes:
* Update how we determine the visibility of the Leave Project button

## How To Test:
* Go to your user page, click on the project options pop, make sure Leave Project shows up when you expect it to and doesn't when it shouldn't.
* Hit the button, make sure you do in fact leave the project. Also check that you get a confirmation dialog for projects you can't automatically rejoin (ie, non-team projects)

## Feedback I'm looking for:
Confirm that my logic is correct? With the change in `leaveProjectDirect` we can't rely on it being undefined for all projects you don't have the permission to delete.
